organization/strategic-council.md

```diff
@@ -0,0 +1,24 @@
+---
+type: Resource
+---
+
+# Strategic council
+
+The strategic council is a group of individual advisors that helps the Foundation For Public Code [board of directors](../organization/governance-model.md#board-of-directors) with key strategic descisions as well as serve as ambasadors of it's [mission](mission.md) and organization.
+
+## Membership of the strategic council
+
+Membership of the strategic council is on invitation from the board of directors.
+
+Members of the strategic council are expected to:
+
+* advise to the board of directors or general assembly on strategic matters
+* serve as a public ambassador for the mission and organization
+* follow along with the briefings sent by the board of directors
+* participate in (email) discussions of the council
+
+This commitment is estimated to be about 15 minutes per week.
```

Comment:
I expect the estimate is expected to be an _average_ weekly amount; if so, might be better to be explicit about that.

Reply:
Added.